### ISSUE TITLE
Google OAuth2 fails with Django 1.11

### ISSUE THREAD

**Issue by JulienFauvel** [python-social-auth/social-app-django#50]
Hello,

while trying to upgrade to the last version of Django, I encountered this error.
I only use Google OAuth2, it works well with Python 3.5.2 and Django 1.10.2.

Here are some infos about the exception thrown :

```
TypeError at /complete/google-oauth2/
unsupported operand type(s) for +: 'NoneType' and 'int'

Django Version: 	1.11
Exception Type: 	TypeError
Exception Value: 	unsupported operand type(s) for +: 'NoneType' and 'int'
Exception Location: 	/opt/local/lib/python3.5/site-packages/social_core/backends/base.py in run_pipeline, line 105
Python Executable: 	/usr/bin/python3
Python Version: 	3.5.2
```

social_core/backends/base.py:105
```python
for idx, name in enumerate(pipeline[pipeline_index:]):
    out['pipeline_index'] = pipeline_index + idx     # Here, values : None + 0
    func = module_member(name)
    result = func(*args, **out) or {}
    if not isinstance(result, dict):
        return result
    out.update(result)
return out
```

Tell me if you need more infos.
Thanks

**Comment by MBryan-QSI** [comment]
Having the same issue since updating django from 1.9 to 1.11.

**Comment by myusuf3** [comment]
@omab same here. on older version can't seem to get a trace to reproduce the issue either locally.

**Comment by omab** [comment]
This commit https://github.com/python-social-auth/social-core/commit/9b2714091be4920ce72fb57bb0ed7f7dc846ce99 should get rid of the problem.

**Comment by knipknap** [comment]
I can confirm that the patch works.

**Comment by it-svit-sbn** [comment]
@omab I believe this issue already fixed in https://github.com/python-social-auth/social-core/commit/9b2714091be4920ce72fb57bb0ed7f7dc846ce99 and released in [1.5.0](https://github.com/python-social-auth/social-core/releases/tag/1.5.0)
